### PR TITLE
Destination test

### PR DESCRIPTION
**Pull request by themonster2015**
This PR adds the following changes to `dev`:

- Create tests for the destination model.

### CODE REVIEW

File: spec/models/destination_spec.rb [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject do
+    Destination.create(name: 'London', location: 'London', price_per_day: 25,
+                       image_url: 'https://via.placeholder.com/150')
+  end
+
+  it 'create a valid destination object' do
+    expect(subject).to be_valid
+  end
+
```
Comment (AleksandarIvezic) [inline spec/models/destination_spec.rb]: Same thing as with Reservation tests. You can remove this piece of code, regarding that you are using shoulda gem.

**Review by AleksandarIvezic — CHANGES_REQUESTED** [review]
Hi @themonster2015,
Great job so far.
However, there are still some things to be improved.
Please check the comments below.

Happy Codding!!! Cheers!!! 🥂

File: spec/models/destination_spec.rb (hunk 1 above)
Comment (themonster2015) [reply spec/models/destination_spec.rb]: Thanks @ShinobiWarior . I've fixed it.

**Comment by danmainah** [comment]
Hi  @themonster2015 

Great job, the tests are running as  they should

Congratulations project Approved ✔✔✔

**Comment by themonster2015** [comment]
> Hi @themonster2015
> 
> Great job, the tests are running as they should
> 
> Congratulations project Approved ✔✔✔

thanks @danmainah

**Pull request merged**